core-model/src/test/java/com/sequenceiq/cloudbreak/TestUtil.java

```diff
@@ -281,6 +281,7 @@ public static InstanceGroup instanceGroup(Long id, String name, InstanceGroupTyp
         instanceGroup.setSecurityGroup(securityGroup(1L));
         instanceGroup.setInstanceGroupNetwork(instanceGroupNetwork());
         instanceGroup.setInstanceMetaData(generateInstanceMetaDatas(nodeCount, id, instanceGroup));
+        instanceGroup.setHints(Set.of("1", "2"));
```

Comment:
May I ask why sometimes we are having `az1` and sometimes it's just `1`? Thank you!!!